microcosm_flask/audit.py

```diff
@@ -182,6 +192,14 @@ def capture_request(self):
             # only capture request body if requested
             return
 
+        if all((
+            request.content_length,
+            self.options.include_request_body is not True,
+            request.content_length >= self.options.include_request_body,
+        )):
+            # don't capture request body if it's too large
```

Comment:
If its json - you can capture the root keys instead

Reply:
If you have a specific algorithm in mind, I'm all ears. I started trying to think about how cleanly truncate a nested JSON document... and represent the fact that it was truncated (e.g. with `...`) and still keep it as valid JSON... and went with the simple thing.